tests/requirements_file_exercise_tests.py

```diff
@@ -0,0 +1,132 @@
+import unittest
+from unittest import mock
+from unittest.mock import call
+
+from src.requirements_file_exercise import create_libraries_string
+
+
+class RequirementsFileExerciseTest(unittest.TestCase):
+    @mock.patch('src.requirements_file_exercise.read_file_lines')
+    def test_should_create_libraries_string_from_single_file(self, mock_read_file_lines):
+        mock_read_file_lines.return_value = ["Flask==2.0.2", "Flask-RESTful==0.3.9", "Jinja2==3.0.2",
+                                             "validators==0.18.2"]
+        file_name = 'path_to_rquirements_file.txt'
+        excepted = "Flask Flask-RESTful Jinja2 validators"
+
+        actual = create_libraries_string(file_name)
+
+        mock_read_file_lines.assert_called_once_with(file_name)
+        self.assertEqual(excepted, actual)
+
+    @mock.patch('src.requirements_file_exercise.read_file_lines')
+    def test_should_create_libraries_string_from_file_with_references(self, mock_read_file_lines):
+        referenced_requirement_file = 'test-requirements.txt'
+        mock_lines_of_first_requirement_file = ["Flask==2.0.2", "Flask-RESTful==0.3.9", "Jinja2==3.0.2",
+                                                "-r " + referenced_requirement_file, "validators==0.18.2"]
+        mock_lines_of_second_requirement_file = ["requests==2.26.0", "six==1.16.0", "typing-extensions==3.10.0.2"]
```

Comment:
In line 23 you call it 'referenced_requirement_file' why now is  'second_requirement_file'?